src/main/java/auth/controller/AppController.java

```diff
@@ -0,0 +1,62 @@
+package auth.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import auth.model.User;
+import auth.service.UserService;
+import auth.validator.UserValidator;
+
+@Controller
+@RequestMapping("/")
```

Comment:
I think will be nicer to have multiple and more specialized controllers: LoginController/RegistrationController/HomePageController etc